services/frontend-service/src/ui/components/dropdown/dropdown.test.tsx

```diff
@@ -0,0 +1,134 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2021 freiheit.com*/
+
+import { DropdownSelect } from './dropdown';
+import { getByTestId, render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { UpdateOverview } from '../../utils/store';
+
+describe('Dropdown', () => {
+    const getNode = (overrides?: {}): JSX.Element | any => {
+        const defaultProps: any = {
+            children: null,
+        };
+        // given
+        return (
+            <div>
+                <MemoryRouter>
+                    <DropdownSelect {...defaultProps} {...overrides} />
+                </MemoryRouter>
+            </div>
+        );
+    };
+
+    const getWrapper = (
+        overrides?: {
+            className: string;
+            applications: any;
+            handleChange: (event: any) => void;
+            isEmpty: (arr: string[] | undefined) => boolean;
+            floatingLabel: string;
+            teams: string[];
+            selectedTeams: string[];
+        },
+        entries?: string[]
+    ) => render(getNode(overrides));
+
+    interface dataT {
+        name: string;
+        className: string;
+        handleChange: (event: any) => void;
+        isEmpty: (arr: string[] | undefined) => boolean;
+        floatingLabel: string;
+        teams: string[];
+        selectedTeams: string[];
+        applications: any;
```

Comment:
I think this `application` is not needed since it's not a prop for DropdownSelect. Also you don't seem to test that value.